testing/regress/ecl/fileservice.ecl

```diff
@@ -33,5 +33,10 @@ SEQUENTIAL(
   File.RenameLogicalFile(prefix + 'afterrename1.d00', prefix + 'scope1::scope2::afterrename2.d00'),
   File.RenameLogicalFile(prefix + 'scope1::scope2::afterrename2.d00', prefix + 'scope1::afterrename3.d00'),
   OUTPUT(DATASET(prefix + 'scope1::afterrename3.d00', rec, FLAT)),
-  File.DeleteLogicalFile(prefix + 'scope1::afterrename3.d00')
+  File.DeleteLogicalFile(prefix + 'scope1::afterrename3.d00'),
+  OUTPUT(ds, , prefix + 'renametest.d00', OVERWRITE),
+  OUTPUT(ds, , prefix + 'afterrename1.d00', OVERWRITE),
+  // Remane with overwrite allowed
+  File.RenameLogicalFile(prefix + 'renametest.d00', prefix + 'afterrename1.d00', true),
+  File.DeleteLogicalFile(prefix + 'afterrename1.d00'),
```

Comment:
Would be good to read the renamed file after the test to verify before deleting it.

Reply:
Added